lib/ui/pages/home_page.dart

```diff
@@ -88,33 +88,39 @@ class _HomePageState extends State<HomePage> {
           items: [1, 2, 3, 4, 5].map((i) {
             return Builder(
               builder: (BuildContext context) {
-                return Column(
-                  children: [
-                    Container(
-                      width: MediaQuery.of(context).size.width,
-                      height: 160,
-                      margin: EdgeInsets.symmetric(horizontal: 5.0),
-                      decoration: BoxDecoration(color: Colors.amber),
-                      child: Container(
-                        child: Text(
-                          'text $i',
-                          style: TextStyle(fontSize: 16.0),
+                return ClipRRect(
+                  borderRadius: BorderRadius.circular(10),
+                  child: Column(
+                    children: [
+                      Container(
+                        width: MediaQuery.of(context).size.width,
+                        height: 160,
+                        margin: EdgeInsets.symmetric(horizontal: 5.0),
+                        decoration: BoxDecoration(
+                          color: AppColor.softGrayColor,
+                        ),
+                        child: Container(
+                          padding: EdgeInsets.all(10),
+                          child: Text(
+                            'Banner $i',
+                            style: TextStyle(fontSize: 16.0),
+                          ),
                         ),
                       ),
-                    ),
-                    Container(
-                      height: 40,
-                      width: MediaQuery.of(context).size.width,
-                      color: Colors.white,
-                      margin: EdgeInsets.symmetric(horizontal: 5.0),
-                      padding: EdgeInsets.only(
-                        top: 8.0,
-                        left: 8,
-                        right: 8,
-                      ),
-                      child: Text("Banner Title $i"),
-                    )
-                  ],
+                      Container(
+                        width: MediaQuery.of(context).size.width,
+                        color: Colors.white,
+                        margin: EdgeInsets.symmetric(horizontal: 5.0),
+                        padding: EdgeInsets.only(
```

Comment:
kalau semua padding 8, bisa di ganti dengan edgeinsets.all(8)

Reply:
Siap, roger that mas